modules/virt-hot-plugging-a-virtual-disk-using-the-web-console.adoc

```diff
@@ -6,16 +6,16 @@
 [id="virt-hot-plugging-a-virtual-disk-using-the-web-console{context}"]
 = Hot-plugging a virtual disk using the web console
 
-Hot-plug virtual disks that you want to attach to a virtual machine instance (VMI) while a virtual machine is running.
+Hot-plug virtual disks that you want to attach to a virtual machine instance (VMI) while a virtual machine is running. When you hot-plug a virtual disk, it remains attached to the VMI until you hot-unplug it.
 
 .Prerequisites
 * You must have a running virtual machine to hot-plug a virtual disk.
 
 .Procedure
 
-. Click *Virtualization* -> *VirtualMachines* from the side menu.
+. Click *Virtualization* from the side menu.
 
-. Select a running virtual machine to open the *VirtualMachine details* page.
+. On the *Virtual Machines* tab, select the running virtual machine to which you want to hot-plug a virtual disk.
 
 . On the *Disks* tab, click *Add disk*.
```

Comment:
I know steps 4 and 5 are not your changes, but we might as well update them to reflect the current UI. 

s/**Add disk**/**Add disk (hot plugged)**

s/Click **Add**/Click **Save**

Reply:
Think I did this as you recommended, but you'll have to check it (as you had said *Add disk* without the parenthetical before. Hopefully users do not have to do both......